ftw/labels/migrations/migrate.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+from ftw.labels.interfaces import ILabelJar
+from ftw.labels.interfaces import ILabelJarChild
+from ftw.labels.interfaces import ILabelRoot
+from ftw.labels.interfaces import ILabelSupport
+from ftw.labels.labeling import ILabeling
+from plone import api
+from zope.annotation.interfaces import IAnnotations
+
+
+# The migration function -------------------------------------------------------
+def migrate_to_2001(context):
+    """
+        Add 'by_user' key to False in jar.
+        Migrate annotation content on ILabelSupport to replace PersistentList by PersistentMapping.
+    """
+    # take all elements who provides ftw.labels.interfaces.ILabelRoot or ILabelJarChild
+    portal_catalog = api.portal.get_tool('portal_catalog')
+    brains = portal_catalog(object_provides=(ILabelRoot.__identifier__, ILabelJarChild.__identifier__))
+    for brain in brains:
+        jar = ILabelJar(brain.getObject())
+        for key in jar.storage.keys():
+            if 'by_user' not in jar.storage[key].keys():
+                # give default value if not exist
+                jar.storage[key]['by_user'] = False
+
+    # take all elements who provides ftw.labels.interfaces.IlabelSupport
+    brains = portal_catalog(object_provides=ILabelSupport.__identifier__)
+    # Transform PersistentList in PersistentMapping
+    for brain in brains:
+        object = brain.getObject()
```

Comment:
Variable names in Python shouldn't shadow builtin names if at all possible. `object` shadows the builtin `object` base class here. Could you please rename it to something like `obj`?